docs/getting_started.mdx

```diff
@@ -8,6 +8,75 @@ tags:
   - Iceberg
 sidebar_position: 2
 ---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+    <TabItem value="s3" label="S3" default>
+# OpenHouse on Spark & S3
+
+In this guide, we will quickly set up a running environment and experiment with some simple SQL commands. Our
+environment will include all the core OpenHouse services such as [Catalog Service](./intro.md#catalog-service),
+[House Table service](./intro.md#house-table-service) and [others](./intro.md#control-plane-for-tables),
+[a Spark 3.1 engine](https://spark.apache.org/releases/spark-release-3-1-1.html) and
+also [MinIO S3 Instance](https://min.io/docs/minio/container/index.html). By the end of this walkthrough, we will have created some tables on OpenHouse,
```

Comment:
In this walkthrough, we will create some tables on Openhouse, insert data in them and query the data.

Reply:
thanks for the quick review!